File: python/cli/prelude_cli/templates/template.c
/*
NAME: $NAME
QUESTION: $QUESTION
CREATED: $CREATED
*/
#include <stdlib.h>
#include <string.h>
#include <stdio.h>

void test(void)
{
    printf("Run test");
    exit(100);
}

void clean(void)
{
    printf("Clean up");
    exit(100);
}

int main(int argc, char *argv[])
{
    if (argc > 1) {
        clean();
    } else {
        test();
    }
}

File: python/cli/prelude_cli/templates/template.rs
/*
NAME: $NAME
QUESTION: $QUESTION
CREATED: $CREATED
*/
fn test() {
    println!("Run test");
    std::process::exit(100);
}

fn clean() {
    println!("Clean up");
    std::process::exit(100);
}

fn main() {
    if std::env::args().len() > 1  {
        clean()
    } else {
        test();
    }
}